### PR TITLE
test: pin MarkupSafe dependency

### PR DESCRIPTION
**Pull request by P403n1x87**
The 2.1.0 release of MarkupSafe comes with some changes that break some
of the projects that depend on it. We fix the framework tests by pinning
the version to the latest 2.0 release for now.

## Checklist
- [ ] Added to the correct milestone.
- [ ] Tests provided or description of manual testing performed is included in the code or PR.
- [ ] Library documentation is updated.
- [ ] [Corp site](https://github.com/DataDog/documentation/) documentation is updated (link to the PR).

### CODE REVIEW

**Review by brettlangdon — COMMENTED** [review]
is this fixed in a newer version of these libraries? should we just bump fastapi from `0.63.0` to `0.75.0` ?

**Comment by P403n1x87** [comment]
> is this fixed in a newer version of these libraries? should we just bump fastapi from `0.63.0` to `0.75.0` ?

Not sure. I could give that a try.

**Review by brettlangdon — APPROVED** [review]

**Review by majorgreys — APPROVED** [review]

File: .github/workflows/test_frameworks.yml [hunk 1]
```diff
@@ -112,21 +112,18 @@ jobs:
...
-      #Reverts SQLalchemy to v1.3.24. Newest version breaks tests due to RowProxy module being deprecated in latest v1.4.11 More info: (https://docs.sqlalchemy.org/en/14/changelog/migration_14.html#rowproxy-is-no-longer-a-proxy-is-now-called-row-and-behaves-like-an-enhanced-named-tuple)
-      - name: Downgrade SQLalchemy
-        run: pip install --force-reinstall SQLAlchemy==1.3.24
       - name: Inject ddtrace
         run: pip install ../ddtrace
       - name: Test
-        run: ddtrace-run pytest tests
+        run: ddtrace-run pytest -p no:warnings tests
 
   flask-testsuite-1_1_4:
```
Comment (Kyle-Verhoog) [inline .github/workflows/test_frameworks.yml:128]: is this still an issue with newer versions of flask? should we just bump the version?

Comment (P403n1x87) [reply .github/workflows/test_frameworks.yml:128]: MarkupSafe 2.1.0 was released on Feb 18, while the last Flask release dates to Feb 14, so I think there might be chances we'd still need this fix at this point in time.

**Review by Kyle-Verhoog — APPROVED** [review]

**Review by brettlangdon — APPROVED** [review]

**Comment by Kyle-Verhoog** [comment]
@mergifyio backport 0.x

**Pull request merged**